src/clients/AcrossConfigStoreClient.ts

```diff
@@ -430,4 +431,27 @@ export class AcrossConfigStoreClient {
     // the chain ID 1.
     return disabledChains.filter((chainId: number) => !isNaN(chainId) && Number.isInteger(chainId) && chainId !== 1);
   }
+
+  async getUBAFeeConfig(
+    chainId: number,
+    token: string,
+    blockNumber: number | "latest" = "latest"
+  ): Promise<UBAFeeConfig> {
+    chainId;
+    token;
+    blockNumber;
+    return new UBAFeeConfig(
+      {
+        default: toBN(0),
+      },
+      toBN(0),
+      {
+        default: [],
+      },
+      {},
+      {
+        default: [],
+      }
+    );
+  }
```

Comment:
This is a placeholder to appease TS

Reply:
Just to clarify, is it necessary to introduce UBA terminology into the ConfigStoreClient? I'm not up to speed exactly on how the config records will be set for UBA, but I'd anticipated that we'd be accessing UBA config via the generic token and global config stores.

Fundamentally I think the lower layers in the stack should avoid  depending on upper layers, but I concede that I don't have the full picture on the requirements for it.